### DISCUSSION TITLE
Anyone having issues with Zigbee2Mqtt ver 1.33.2?

### DISCUSSION THREAD

**Discussion by djcrawleravp** [basnijholt/adaptive-lighting#829] [Q&A]
After yesterday's update of Zigbee2Mqtt (version 1.33.2), I've been experiencing an issue where every time I turn on my lights, they remain in the last state, and the adaptive lighting feature no longer activates, or sometimes it takes several minutes to do so.

I reverted back to version 1.33.1 of Zigbee2Mqtt, and everything is working smoothly again.

**Comment by fbloemhof** [comment]
I saw the same and found #506, seems that setting these config fixed fixed this issue.

**Comment by djcrawleravp** [comment]
Thanks @fbloemhof !

It seems it's working now some lights slower and some even faster (i'll give them a day to test)

Just curious, do you know what is this? It keeps appearing on the logs now

<img width="1249" alt="Screen Shot 2023-12-07 at 09 42 10" src="https://github.com/basnijholt/adaptive-lighting/assets/12896341/8a849d34-683c-4814-8ede-1b5d5f088747">

Thanks again!!

**Reply by fbloemhof** [reply]
I don't see those errors, sorry.
